test/test-applications/integrations/TestApplication.NServiceBus/TestApplication.NServiceBus.csproj

```diff
@@ -6,8 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
     <PackageReference Include="NServiceBus" VersionOverride="$(LibraryVersion)" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" />
```

Comment:
Why changes in csproj files are needed?
I would expect that loading library process will be not changed.

BTW NServiceBus tests are consistently failing.

Reply:
Strange that NServiceBus is failing after I fixed it...the issue is that the dependency loading did somehow change for loading dependent libraries, though I think it was only for .NET Framework. I can't explain exactly how at the moment, but the side-effect was that there were duplicate assemblies in process for the DependencyInjection and DiagnosticSource until the application brought the correct copy.

Reply:
NServiceBus passed on GH, also I have executed in several times on my Windows machine, always passing.